src/redux/drawer/drawerSlice.ts

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from "@reduxjs/toolkit"
+import type { RootState } from "../store"
+
+export const initialState = {
+  drawer: false,
```

Comment:
💭 Consider using a predicate with an adjective (e.g. `isOpen`). Drawer is the context in this case, so it's not necessary to include it as a property. For example—`state.drawer.isOpen` reads better than `state.drawer.drawer`.